kafka/record/memory_records.py

```diff
@@ -166,7 +166,7 @@ def size_in_bytes(self):
 
     def compression_rate(self):
         assert self._closed
-        return self.size_in_bytes() / self._bytes_written
+        return self.size_in_bytes() / float(self._bytes_written)
```

Comment:
This is only required for python2; python3 division returns a float by default. I think a cleaner way to handle this compatibility is with `__future__`:
```
from __future__ import division
```

With that you shouldn't need to cast to float